### PR TITLE
Console - Move fonts to public folder

### PR DESCRIPTION
**Pull request by Vampouille**
So we don't need to allow a new path on security-proxy

### CODE REVIEW

File: console/src/main/webapp/manager/app/assets/fonts [hunk 1]
```diff
@@ -0,0 +1 @@
+../../fonts/
```
Comment (tonio) [inline console/src/main/webapp/manager/app/assets/fonts]: Why still using this fonts dir, and not symlink directly to bootstrap's one in node_modules?

File: console/src/main/webapp/manager/app/styles/app.less [hunk 2]
```diff
@@ -5,6 +5,13 @@
 @tablet: 768px;
 @large: 992px;
 
+@font-face {
+  font-family: 'Glyphicons Halflings';
+
+  src: url('./fonts/glyphicons-halflings-regular.eot');
+  src: url('./fonts/glyphicons-halflings-regular.eot?#iefix') format('embedded-opentype'), url('./fonts/glyphicons-halflings-regular.woff2') format('woff2'), url('./fonts/glyphicons-halflings-regular.woff') format('woff'), url('./fonts/glyphicons-halflings-regular.ttf') format('truetype'), url('./fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular') format('svg');
+}
+
```
Comment (tonio) [inline console/src/main/webapp/manager/app/styles/app.less]: Did you check there was no HTTP requests to the path defined in bootstrap?

**Comment by fvanderbiest** [comment]
@Vampouille up

**Comment by tonio** [comment]
Ready for re-review :sunglasses:

**Pull request merged**